infraohjelmointi_api/migrations/0017_budgetgroup_constructionphase_planningphase_and_more.py

```diff
@@ -0,0 +1,221 @@
+# Generated by Django 4.1.3 on 2022-12-22 09:10
+
+import django.core.validators
+from django.db import migrations, models
+import django.db.models.deletion
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("infraohjelmointi_api", "0016_rename_riskassess_project_riskassessment"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="BudgetGroup",
```

Comment:
If this is meant to store Project Classes then it should be renamed to ProjectClass. Also it must have parent, and path fields